### PR TITLE
Add unit tests for handleRecaptchaFlow

### PR DESCRIPTION
**Pull request by renkelvin**
Unit tests for https://github.com/firebase/firebase-js-sdk/pull/7666

### CODE REVIEW

**Comment by renkelvin** [comment]
@prameshj please take a look.

File: packages/auth/src/platform_browser/recaptcha/recaptcha_enterprise_verifier.test.ts [hunk 1]
```diff
@@ -117,4 +126,109 @@ describe('platform_browser/recaptcha/recaptcha_enterprise_verifier', () => {
...
+        mockAuthInstance,
+        mockRequest,
+        RecaptchaActionName.SIGN_IN_WITH_PASSWORD,
+        mockActionMethod
+      );
+      expect(mockActionMethod).to.have.been.calledOnce;
+      expect(response).to.equal('testResponse');
+    });
+
+    it('should handle MISSING_RECAPTCHA_TOKEN error when emailPasswordEnabled is false', async () => {
```
Comment (prameshj) [inline packages/auth/src/platform_browser/recaptcha/recaptcha_enterprise_verifier.test.ts:167]: This error will be handled even if "emailPasswordEnabled" is true or if the config does not exist too, right?

Maybe add a comment? 

"Errors like "MISSING_RECAPTCHA_TOKEN" will be handled irrespective of the enablement status of "emailPasswordEnabled", but this test verifies the more likely scenario where emailPasswordEnabled is false"

**Review by prameshj — APPROVED** [review]
Thanks for adding this!

File: packages/auth/src/platform_browser/recaptcha/recaptcha_enterprise_verifier.test.ts (hunk 1 above)
Comment (renkelvin) [reply packages/auth/src/platform_browser/recaptcha/recaptcha_enterprise_verifier.test.ts:167]: Yeah, it's not necessarily "emailPasswordEnabled" is false. Added the comment.

**Comment by renkelvin** [comment]
Hey @hsubox76 , could you help take a look? Thanks!

**Review by hsubox76 — APPROVED** [review]
Since it's just unit tests and doesn't need a release, I don't think it needs a changeset.

**Pull request merged**